clientmanager/condor/starter.py

```diff
@@ -78,7 +79,8 @@ def make_condor_job_description(
         "transfer_executable": "false",
         #
         "request_cpus": str(n_cores),
-        "request_memory": memory,
+        "request_memory": str(worker_memory_bytes),
+        "request_disk": str(worker_disk_bytes),
```

Comment:
This doesn't do what you think it does :frowning_face:. If you don't specify a unit, condor uses MB for memory and KB for disk.